bindings/storage.i

```diff
@@ -95,6 +95,7 @@
 %include "../../storage/Utils/HumanString.h"
 %include "../../storage/Utils/Region.h"
 %include "../../storage/Utils/Remote.h"
+%include "../../storage/Utils/Logger.h"
```

Comment:
is there any ordering rule for includes? for alphabetical one it should be a bit upper